wiki/People/Users_with_unique_titles/en.md

```diff
@@ -189,6 +189,15 @@ The osu!remix contests award the **osu!mixer** title as a first-place prize.
 
 ¹ ![][flag_US] [BilliumMoto](https://osu.ppy.sh/users/3862471) was awarded with the **osu!mixer II** title instead for having won both the [osu!remix contest #1](https://osu.ppy.sh/home/news/2016-08-29-osuremix-contest-1-results) and the [A Labour of Love](/wiki/Contests/A_Labour_of_Love_Contest) contest.
 
+## Current osu!trivium Quiz contest winners
+
+The osu!trivium Quiz contests award its winning team or individual the **osu!trivium Champion** title, which lasts for one year. These are the title holders for 2021:
```

Comment:
I'd plug a link to a relevant newspost somewhere here

Reply:
would be more appropriate in its own contest article

Reply:
there was also no results news post, just a livestream (and no official tweet even)

Reply:
well, having a link even to an opening newspost would be good imo